lib/src/records/property_change_record.dart

```diff
@@ -26,6 +26,20 @@ class PropertyChangeRecord<T> implements ChangeRecord {
   );
 
   @override
+  bool operator ==(Object o) {
+    if (o is PropertyChangeRecord<T>) {
```

Comment:
[nit] I think this looks a bit nicer if we consistently choose '&&' instead of pulling one of the pieces into an if block. It even formats nicely :)

    bool operator ==(Object o) =>
        o is PropertyChangeRecord<T> &&
        identical(object, o.object) &&
        name == o.name &&
        oldValue == o.oldValue &&
        newValue == o.newValue;